### PR TITLE
1 Add database schema

### PR DESCRIPTION
**Pull request by MaxShelestiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,3 +77,6 @@ This is a list of the required environment variables:
 #### MSSQL Server
 
 **SA_PASSWORD** - MSSQL Server "SA" user password
+
+## SQL Server Database Schema
+![Database_Schema](Squirrel_DB_Schema.svg)
```
Comment (Anatolii-Ivanov) [inline README.md]: Either use Mermaid for it or find a way to update svg file and add to documentation)

Comment (ABertashDev) [reply README.md]: DB schema updated with Mermaid

**Review by alinaSielina — COMMENTED** [review]
Looks good, but I have a few clarifications:
- UserSettings and ProjectSettings tables. I'm not entirely sure about the purpose of the SettingsUrl column. Given the relatively simple addition, I suggest we consider including these tables later, perhaps when we implement features such as the settings page. Their inclusion should not greatly affect the overall structure.
- The GroupsTags, Groups and UserGroups tables are redundant. Instead, we can update the UserProjects table by adding a Role(int) column and then later include the RoleEnum in our code.
- Branch table - I understand the IsActive column will be used for soft delete? We decided to implement a soft delete function, but maybe we should call it IsDeleted?
- Comments table - we will use comments not only for PRs, so I think we should now expand this table, and it could look like this for example:
   Id (int), 
   AuthorId (int), 
   Content (nvarchar(500)), 
   CreatedAt (datetime), 
   UpdatedAt (datetime), 
   CommentableId (int), 
   CommentableType (int)
   
- ProjectFiles table. I don't fully understand why this table refers to Users and Projects and not Commits. Or maybe you added this table for the README files? So I think we can use a Git-like structure:
   
image

image

something like this:
image

File: README.md [hunk 2]
```diff
@@ -77,3 +77,182 @@ This is a list of the required environment variables:
...
+        bit IsDefault
+    }
+
+    Users {
+        int Id PK
+        nvarchar(25) UserName
+        nvarchar(25) FirstName
+        nvarchar(25) LastName
+        nvarchar(50) Email
+        nvarchar(25) Password
```
Comment (OlegKarapysh) [inline README.md]: it is bad to store user's password explicitly, so maybe we should hash it with salt. In this case User entity should have 'Salt' attribute of nvarchar type

**Review by OlegKarapysh — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ABertashDev) [reply README.md]: Thanks. Added Salt column and changed Password column to PasswordHash

**Review by alinaSielina — APPROVED** [review]

**Pull request merged**